packages/core/tests/unit/models/user-test.js

```diff
@@ -130,4 +130,16 @@ module('Unit | Model | user', function(hooks) {
       assert.equal(rules.get('firstObject'), deliveryRule, 'user deliveryRule property contains deliveryRule model');
     });
   });
+
+  test('add role to user', async function(assert) {
```

Comment:
Since this is basically the same test as the delivery rules test above it
```suggestion
  test('roles relationship', async function(assert) {
```